content_schemas/formats/shared/definitions/content_id_alias.jsonnet

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do I need to move the addition of this new field into its own PR as per [the documentation here](https://github.com/alphagov/publishing-api/blob/main/docs/content_schemas/changing-an-existing-content-schema.md#general-workflow) or is it ok to sit within this PR?

Reply:
I'd leave it within this PR, as the schema change is related to the other commits.